NuGet.Config

```diff
@@ -1,8 +1,8 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <configuration>
   <packageSources>
-     <!-- To enable LocalFeed for testing uncomment the following line -->
-     <add key="NugetOfficialV3" value="https://api.nuget.org/v3/index.json" />
-     <add key="Local" value="tools/LocalNugetFeed" />
-   </packageSources>
+	<!-- To enable LocalFeed for testing uncomment the following line -->
```

Comment:
@anusapan what is the need to edit this file? All this is doing is deleting spaces. Can you revert this change.

Reply:
reverted